contracts/interfaces/IPrtStakingPool.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: Apache License, Version 2.0
+pragma solidity ^0.6.10;
+
+interface IPrtStakingPool {
+    function accrue(uint256 _amount) external;
+    function feeSplitExtension() external view returns (address);
```

Comment:
Noting the newer version of SnapshotStakingPool uses the variable name `distributor` instead of `feeSplitExtension`

Reply:
fixed here [`515e783`](https://github.com/IndexCoop/index-coop-smart-contracts/pull/176/commits/515e7838fad7accf225377ebff80ad194cfdac93)